src/commentscan.l

```diff
@@ -3262,7 +3262,9 @@ static bool handleSection(yyscan_t yyscanner,const QCString &s, const StringVect
   else if (s=="subparagraph")    yyextra->sectionLevel=SectionType::Subparagraph;
   else if (s=="subsubparagraph") yyextra->sectionLevel=SectionType::Subsubparagraph;
   // raise it if requested
-  yyextra->sectionLevel = std::min(yyextra->sectionLevel + yyextra->raiseLevel,SectionType::MaxLevel);
+// paule32 - msvc problem with std::min
+// yyextra->sectionLevel = std::min(yyextra->sectionLevel + yyextra->raiseLevel,SectionType::MaxLevel);
+  yyextra->sectionLevel = min(yyextra->sectionLevel + yyextra->raiseLevel,SectionType::MaxLevel);
   // rewrite the update section level to the output
```

Comment:
- What kind of problems with the compiler
- Which version of the compiler

Reply:
the Linux Compilers for Ubuntu Action's takes care, and they work - I thinking so, because I have 6 Workflows running well. The other Workflows for Windows crashed when you would use **std::min**.
I think there is a missing Header File for it.
I don't work with MSVC, so I can not say what the problems is, directly.
Maybe a missing header - or **using namespace std;**
I don't compile it on my Computers, it is the Workflow from Doxygen GitHub Account, that I had forked.
I run Action's directly under GitHub - it takes some minutes, but it is possible.
Because, I download a archive of Doxygen last days, and tried to Compile under Mingw64 Bit (MSYS2) under Windows, and getting a Bison 3 problem, so the build CMakefile was created, but the make Makfile produce error, so I decide to compile under GitHub - where I can test for Windows, Linux, and MacOS in (nearly) one steps.

**BTW**: You noticed me about the yml File... I delete it on my repro - I hope it is all well.